negotiation/contract.negotiation.protocol.md

```diff
@@ -39,6 +39,21 @@ The CN state machine is represented in the following diagram. Note that transiti
+| OFFERED                    | C      | ContractRequestMessage                    | REQUESTED    | The consumer declines the offered contract and might sent a counter offer to the provider.                                                                                                                                                                                                                                               |
+| OFFERED                    | C      | ContractNegotiationEventMessage:accepted  | ACCEPTED     | The consumer accepts to comply with the defined details in the contract.                                                                                                                                                                                                                                                                 |
+| ACCEPTED                   | P      | ContractAgreementMessage                  | AGREED       | The provider sends a provider-signed contract agreement to the consumer.                                                                                                                                                                                                                                                                 |
+| AGREED                     | C      | ContractAgreementVerificationMessage      | VERIFIED     | The consumer sends a both-sides signed contract agreement to the provider.                                                                                                                                                                                                                                                               |
+| VERIFIED                   | P      | ContractNegotiationEventMessage:finalized | FINALIZED    | The provider assures the consumer to have received the signed agreement.                                                                                                                                                                                                                                                                 |
+| \*/{TERMINATED, FINALIZED} | {C, P} | ContractNegotiationTerminationMessage     | FINALIZED    | The provider or consumer is not willing to continue any further negotiation process. This might happen e.g. in the case the provider doesnt want to place any offer (i.e. data) to the consumer.                                                                                                                                         |
```

Comment:
`TERMINATED` and `FINALIZED` are never initial states. The table may contain the additional transitions introduced in #43 either in the same rows than the success states, or new ones.

Reply:
```suggestion
| * \ {TERMINATED, FINALIZED} | {C, P} | ContractNegotiationTerminationMessage     | FINALIZED    | The provider or consumer is not willing to continue any further negotiation process. This might happen e.g. in the case the provider doesnt want to place any offer (i.e. data) to the consumer.                                                                                                                                         |
```